### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by Magenent** [EarthMC/Issue-Tracker#145]
Terra Nova

For some reason when I enchant with lvl 30 enchants it only enchants with what the hint is instead of also some hidden stuff idk if its the lag but it's bad.

1.Make a level 30 enchantment table
2. Right click on it
3. Try to enchant something with the lvl 30 enchant
4. You should see the glitch

It should give some random enchants but it doesn't.

**Comment by Arganog** [comment]
I have not noticed this myself, and I've been enchanting today.

Perhaps you just got unlucky?

**Comment by Magenent** [comment]
> I have not noticed this myself, and I've been enchanting today.
> 
> Perhaps you just got unlucky?

yeah it was just because of the lag, cuz after the restart it started working
